src/tools/roc_conv/main.cpp

```diff
@@ -37,6 +38,23 @@ int main(int argc, char** argv) {
     core::Logger::instance().set_level(
         LogLevel(core::DefaultLogLevel + args.verbose_given));
 
+    switch ((unsigned)args.color_arg) {
+    case color_arg_auto:
+        if (core::colors_available())
+            core::Logger::instance().set_colors(core::ColorsMode(core::ColorsEnabled));
+        break;
+
+    case color_arg_always:
+        core::Logger::instance().set_colors(core::ColorsMode(core::ColorsEnabled));
+        break;
+
+    case color_arg_never:
+        break;
```

Comment:
Nitpick: I suggest to explicitly set the color mode to Disabled here to avoid relying on the specific default value.